qa/common/infer_util.py

```diff
@@ -896,12 +903,16 @@ def infer_zero(tester,
         platform = metadata.platform
 
     for io_num in range(io_cnt):
-        if platform == "pytorch_libtorch":
-            input_name = "INPUT__{}".format(io_num)
-            output_name = "OUTPUT__{}".format(io_num)
+        if override_input_names:
+            input_name = override_input_names[io_num]
+            output_name = override_output_names[io_num]
```

Comment:
Should this be under condition override_output_names ?

Reply:
Great point. Fixed.